content/docs/reference/config/helm-customization/index.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Customise the Helm Installation
+description: Customise the Istio installation with the included Helm chart.
+weight: 30
+keywords: [kubernetes,helm]
+aliases:
+    - /docs/reference/config/helm/index.html
+---
+
+The Helm chart ships with reasonable defaults.  There may be circumstances in which defaults require overrides.
+To override Helm values, use `--set key=value` argument during the `helm install` command.  Multiple `--set` operations
+may be used in the same Helm operation.
+
+Helm charts expose configuration options which are currently in alpha.  The currently exposed options are explained in the
+following table:
+
+<!-- AUTO-GENERATED-START -->
+| Parameter | Default | Description | Values |
```

Comment:
Given the Helm command-line says "--set key=value", then should "Parameter" be called "Key" instead?

It doesn't seem as though the Values column should be using `` on the listed values. In fact, looking at the generated table, it doesn't appear as though the Values column is useful at all. There's a Description column which seems sufficient.

Reply:
Will update the python script as well